enterprise/backend/src/metabase_enterprise/sso/api/sso.clj

```diff
@@ -43,7 +43,7 @@
                  :exceptionClass (.getName Exception)
                  :additionalData data}))})
 
-(api/defendpoint POST "/"
+(api/defendpoint ^{:content-types #{:content/form}} POST "/"
```

Comment:
Are you sure this should only be `:content/form`? This is the endpoint we get in the SSO response from the IdP... not sure what `Content-Type` they're sending us but I would have guessed there would be a good chance it could be JSON

Reply:
i can add both. Or perhaps this could be `:content/*`. I have no hope that this will hit the release and should bake in master for a while. I'm also considering an env var to turn it off in case it proves unworkable in some cases.

Reply:
I think `:content/*` makes sense here. I don't think we need an env var for this -- we really don't support anything besides `Content-Type: application/json` for anything except for a handful of endpoints. An env var to disable the error wouldn't change that fact. I like keeping it required and giving it time in `master` to bake

Reply:
The issue is happening in CI. Lots of things don't send a content type at all. I'm making a change that if there are no bindings for the route other than route-provided bindings (card-id from the route itself) then it does not enforce the content type. But its possible lots of api usage (or maybe our browser client?) won't set this. Seemed safer to be able to disable it if it proved buggy at first.